### ISSUE TITLE
Uncaught (in promise) Error: multiple nodes output to one data value

### ISSUE THREAD

**Issue by huangyakun** [Microsoft/onnxjs#16]
When I load a pytorch export onnx model, something wrong happend like this, what's wrong with my operations? I have checked the exported onnx model with onnx.checker. Thanks!
graph.ts:298 Uncaught (in promise) Error: multiple nodes output to one data value: 19
    at t.buildGraph (graph.ts:298)
    at new t (graph.ts:139)
    at Object.from (graph.ts:77)
    at t.load (model.ts:29)
    at session.ts:85
    at t.event (instrument.ts:294)
    at e.initialize (session.ts:81)
    at e.<anonymous> (session.ts:63)
    at onnx.ts:87
    at Object.next (onnx.ts:87)

**Comment by fs-eire** [comment]
thanks for the feedback. could you please share the model so that I can take a look?

**Comment by huangyakun** [comment]
Hi,
     Thanks for the feedback. 
      You can download the onnx from 
      https://drive.google.com/open?id=1kXdkxmyhUpnL_-T5K8-Wb5CyQjTnZc1V
     And another onnx model has the same problem which can be downloaded from
      https://drive.google.com/open?id=1N6eDfXDE-9QmFOOemC9FkjqC_D1NUWk8
Thanks!
Huang

**Comment by hariharans29** [comment]
@fs-eire - For the second issue (operator BatchNormalization reports 'InvalidInputs' because of X.dims.length < 3), the input data X should be atleast 3 dimensional. The first dimension is batch size, the second is channels, and the third onwards are the data dimensions. More details [here](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#inputs-9).

**Comment by ybrnathan** [comment]
Double checked PyTorch BatchNormalization implementation, seems like PyTorch has two versions:
- torch.nn.BatchNorm1d
    Input: (N,C) or (N,C,L)
    Output: (N,C) or (N,C,L)

- torch.nn.BatchNorm2d
    Input: (N,C, H, W)
    Output: (N,C, H, W)

And according ONNX spec, ONNX only support (N, C, H, W) or (N, C, L) and does not support (N, C). In your model, The BatchNormalization node after MatMul input shape is (N, C) which is not supported according to ONNX spec.

**Comment by hariharans29** [comment]
Closing this issue as we root-caused the issue. ONNX spec doesn't support BatchNormalization op taking input of shape (N,C)
